transformer_engine/pytorch/csrc/extensions.cu

```diff
@@ -1131,23 +1131,24 @@ at::Tensor fp8_transpose(at::Tensor input,
 }
 
 
-at::Tensor fp8_gelu(at::Tensor input,
-                    at::Tensor scale,
-                    at::Tensor amax,
-                    at::Tensor scale_inv,
-                    transformer_engine::DType otype
+at::Tensor gelu(at::Tensor input,
+                at::Tensor scale,
+                at::Tensor amax,
+                at::Tensor scale_inv,
+                transformer_engine::DType otype
 ) {
   using namespace transformer_engine;
 
-  size_t M = static_cast<size_t>(input.size(0));
-  size_t N = static_cast<size_t>(input.size(1));
+  size_t N = static_cast<size_t>(input.size(-1));
+  size_t M = numel(input) / N;
```

Comment:
Is the `numel` function defined? If not:
```suggestion
  size_t M = input.numel() / N;
```
I see a few other `numel` s in this file.